pages/common/coffee.md

```diff
@@ -0,0 +1,23 @@
+# coffee
+
+> Executes CoffeeScript scripts or compiles them into JavaScript.
+
+- Run script:
+
+`coffee {{filename}}.coffee`
+
+- Compile to JavaScript and save file:
+
+`coffee --compile {{filename}}.coffee`
+
+- Compile to JavaScript and save as file:
+
+`coffee --compile {{filename}}.coffee --output {{filename}}`
+
+- Run interactive REPL:
+
+`coffee --interactive`
+
+- Watch script for changes and rerun:
```

Comment:
What is meant by rerun here?

Reply:
What this does is 1. run the script, 2. watch for changes and 3. run again if it changed.